src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.card {
+  &:nth-child(-n + 4) {
+    margin-bottom: $gap;
+  }
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.